prov/gni/src/gnix_ep.c

```diff
@@ -113,21 +113,31 @@ static inline ssize_t __ep_recv(struct fid_ep *ep, void *buf, size_t len,
 
 static inline ssize_t __ep_recvv(struct fid_ep *ep, const struct iovec *iov,
 				 void **desc, size_t count, fi_addr_t src_addr,
-				 void *context, uint64_t flags, uint64_t tag,
-				 uint64_t ignore)
+				 void *context, uint64_t flags, uint64_t ignore,
+				 uint64_t tag)
 {
 	struct gnix_fid_ep *ep_priv;
 
-	if (!ep || !iov || count != 1) {
+	if (!ep || !iov || !count) {
 		return -FI_EINVAL;
 	}
 
+/* if (count == 1) { */
+/* return _gnix_recv(ep_priv, (uint64_t)iov[0].iov_base, */
+/* iov[0].iov_len, desc ? desc[0] : NULL, */
+/* src_addr, context, */
+/* ep_priv->op_flags | flags, tag, ignore); */
+/* } */
```

Comment:
Turning recvv(count 1) into recv seems like a good idea.  Any problem enabling this?